.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+compute_engine_instance:
+  image_project: freebsd-org-cloud-dev
+  image: family/freebsd-13-1
+  platform: freebsd
+  disk: 100 # Gb
+
+build_task:
+  name: build
+  timeout_in: 120m
+  only_if: $CIRRUS_TAG != ''
+  env:
+    AWS_ACCESS_KEY_ID: ENCRYPTED[25d5e1d29570da70ceeb75e842b98be63d110824997b3c6b7bc0735b8acc686fbe0e9995cfe4d22a61d707a2e705d48b]
+    AWS_SECRET_ACCESS_KEY: ENCRYPTED[7fc23de7b958e671dcb485ff0baef777b83fb472f0afdde940122c994bc1586710d69b9d25855768bfd201904f3654dd]
+    S3_HOST: ENCRYPTED[d3fef1b5850e85d80dd1684370b53183df2218f2d36509108a2703371afd9ebd3f9596ad4de52487c15ea29baed606b7]
+    TARBALL_EXT: "tar.xz"
+    ARCH: 64
+    ARTIFACT: "x86_64-freebsd"
+    DISTRO: "na"
+    RUNNER_OS: "FreeBSD"
+    ADD_CABAL_ARGS: "--enable-split-sections"
+    GITHUB_WORKSPACE: ${CIRRUS_WORKING_DIR}
+  install_script: pkg install -y hs-cabal-install git bash misc/compat10x misc/compat11x misc/compat12x gmake patchelf tree
+  script:
+    - tzsetup Etc/GMT
+    - adjkerntz -a
+    - bash .github/scripts/build.sh
+    - bash .github/scripts/test.sh
+      # binaries_cache: &binaries_cache
+      #   folder: out
+      #   fingerprint_script:
+      #     - ghc --numeric-version
+      #     - cat cabal.project
+      #     - cat haskell-language-server.cabal
+      #     - cat dist-newstyle/cache/plan.json
+  binaries_artifacts:
+    path: "out/*"
+
+    # release_task:
+    #   name: "Release"
```

Comment:
why is this commented out?

Reply:
I couldn't make it work for now (pushing the release artifact of Cirrus CI to github release). I leave it commented out so I can return on it later without digging through the same documentation or git history.